src/metabase/models/collection.clj

```diff
@@ -1237,37 +1237,6 @@
                              not-trash-clause
                              (or where true)]}))))
 
-(defmethod serdes/extract-one "Collection"
-  ;; Transform :location (which uses database IDs) into a portable :parent_id with the parent's entity ID.
-  ;; Also transform :personal_owner_id from a database ID to the email string, if it's defined.
-  ;; Use the :slug as the human-readable label.
-  [_model-name _opts coll]
-  (let [fetch-collection    (fn [id]
-                              (t2/select-one Collection :id id))
-        {:keys [parent_id]} (some-> coll :id fetch-collection (t2/hydrate :parent_id))
-        parent              (some-> parent_id fetch-collection)
-        parent-id           (when parent
-                              (or (:entity_id parent) (serdes/identity-hash parent)))
-        owner-email         (when (:personal_owner_id coll)
-                              (t2/select-one-fn :email 'User :id (:personal_owner_id coll)))]
-    (-> (serdes/extract-one-basics "Collection" coll)
-        (dissoc :location)
-        (assoc :parent_id parent-id
-               :personal_owner_id owner-email)
-        (assoc-in [:serdes/meta 0 :label] (:slug coll)))))
```

Comment:
oh interesting what is this weird thingie...

Reply:
I don't know why this was done manually, but the same property is getting set by `serdes/generate-path` now.

Reply:
I guess one difference is that doing it manually skipped calling slugify again on the value. I'd assume that something called `:slug` is already a safe value, and that would be a no-op.